tests/python/unittest/test_sparse_ndarray.py

```diff
@@ -638,6 +638,17 @@ def test_create_row_sparse():
         rsp_copy = mx.nd.array(rsp_created)
         assert(same(rsp_copy.asnumpy(), rsp_created.asnumpy()))
 
+        # add this test since we added np.int32 and np.int64 to integer_types
```

Comment:
@eric-haibin-lin I added this unit test for the function since `integer_types` now includes `np.int32` and `np.int64`.